README.md

```diff
@@ -2,35 +2,47 @@
   <br>
   <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
   <br>
-  JS Parser
+  JavaScript Parser
 </h5>
-<h4 align="center">Parse and validate AsyncAPI documents</h4>
+<p align="center">
+  <em>Use this package to parse and validate AsyncAPI documents —either YAML or JSON— in your Node.js or browser application. Updated bundle for the browser is always attached to the GitHub Release.</em>
+</p>
 
----
+![npm](https://img.shields.io/npm/v/@asyncapi/parser?style=for-the-badge) ![npm](https://img.shields.io/npm/dt/@asyncapi/parser?style=for-the-badge)
 
-## :loudspeaker: ATTENTION:
+<br><br>
```

Comment:
Why double `<br>` instead of a heading? I think we're losing clarity here.

![Screenshot 2020-08-11 at 11 16 54](https://user-images.githubusercontent.com/242119/89880191-42dbde80-dbc4-11ea-8f72-847eb3fee460.png)

Also, the AsyncAPI 1.x notice feels very attached to the ToC which is completely a different section. What about moving the notice **after** the ToC and adding a heading saying "Table of Contents" or similar?

Reply:
ToC is there so you do not go below ToC but can just jump to a specific section. I've put it there as this is where we have notice in Generator readme, same with `<br>`. I also want to avoid writing `table of contents`, don't like to write obvious things

if I remove those brs, I think it looks better

<img width="1024" alt="Screenshot 2020-08-11 at 14 37 18" src="https://user-images.githubusercontent.com/6995927/89898027-48dfb880-dbe0-11ea-9cc3-68f7882ea6d5.png">